.github/workflows/c-cpp.yml

```
@@ -6,166 +6,78 @@ on:
+          - macos
+        include:
+          - os: linux
+            distro: ubuntu
+            release: 24.04
+            msvc: true
+          - os: macos
+            distro: macos
+            release: 15
     steps:
-    - name: Add apt.llvm.org
+    - name: Upgrade packages and install Clang 21
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)